index.js

```diff
@@ -1,1144 +0,0 @@
-  console.log(
-    "\nA large and ancient bookshelf looms before you, filled with numerous volumes of books, games, and little treasures. You pick the first book that catches your eye, and... what.. no way! A signed copy of The Return of the King?! For a moment, you consider tucking this away for your personal collection... but, sigh- if someone sees you might lose your investigator's license. Ah well, you can dream. Seriously though, how did Mr. Green acquire that...?"
-  );
-});
-
-let libFireplace = new Item("Fireplace", "A fireplace", false, () => {
-  console.log(
-    "\nYou gaze at the warm, crackling fireplace. Who knows who's been feeding the fire, but it's orange glow illuminates the book case and casts all sorts of furtive shadows across an otherwise dark room (no windows, poor design, really). Picking up a piece of wood, you add it the pile, causing the flame to briefly grow a little higher. A glimpse of hope in the dark. "
-  );
-});
-
-let libSofa = new Item("Sofa", "A sofa", false, () => {
-  console.log(
-    "\nAha! A clubby, tufted leather Chesterfield. Though well worn from ages of use, it's a handsome sofa that strikes a bold silhouette and commands respect. Situated across from the fire, it would be a lovely place to read or take a nap... *you sit down and gaze longingly into the fire for a moment* *10 minutes pass* Hmmmm, you could easily got lost in a trance here... best to move on!"
-  );
-});
-
-// Billiard Room inventory ////////////////////////////////////////////////////////////////////////
-
-let billiardTable = new Item(
-  "Billiard Table",
-  "A billiard table.",
-  false,
-  () => {
-    console.log(
-      `\nYou set your eyes upon a luxuriously built billiard table with what appear to be beautifully hand-crafted leather pockets.`
-    );
-  }
-);
-
-let billiardBalls = new Item(
-  "Billiard Balls",
-  "A set of beautiful billiard balls.",
-  false,
-  () => {
-    console.log(
-      `\nYou look upon the table to find billiard balls scattered across the table as if a game in progress had been abandoned. As you count up the balls you notice that one is missing...`
-    );
-  }
-);
-
-let poolCue = new Item(
-  "A billiard cue",
-  "A nice, but unremarkable pool cue.",
-  false,
-  () => {
-    console.log(`\nA nice, but unremarkable pool cue.`);
-  }
-);
-
-// Room entry messages ////////////////////////////////////////////////////////////////////////////
-
-const studyMessage = "\nYou have entered the Study.";
-
-const billiardMessage = "\nYou have entered the Billiard Room.";
-
-const conservatoryMessage = "\nYou have entered the Conservatory.";
-
-const diningMessage = "\nYou have enetered the Dining Room.";
-
-const libraryMessage = "\nYou have entered the Library.";
-
-const kitchenMessage = "\nYou have entered the Kitchen.";
-
-const ballroomMessage = "\nYou have enetered the Ballroom.";
-
-const loungeMessage = "\nYou have entered the Lounge.";
-
```

Comment:
These could have been added as a part of the room objects either as part of the description or as a new entry on the object

Reply:
Agreed, I will change them.

Reply:
Changed